src/bluefin_v2_client/client.py

```diff
@@ -528,6 +534,7 @@ async def adjust_margin(
                 "exchange",
                 self.account.getUserAddress(),
                 self.contracts.get_package_id(),
+                typeArguments=[self.contracts.get_currency_type()],
```

Comment:
Is this change backward compatible ?

If not do you think we should do this change separately because the retries thing i see more of bug fix and If clients complain use we can ask them to point to the version with retries

Reply:
Yes, what i will do is that i will create a new pr and add changes for rpc.py there and create pr